EMCC_DEBUG=1 throws error

Hi! My environment:
```
Node v8.0.0
emcc (Emscripten gcc/clang-like replacement) 1.38.23 (commit 443447b14d34a3d710d33f229dea64e5c78a78ce)
Ubuntu 18.04.1 LTS
```

I can built my project without errors, but if I use EMCC_DEBUG=1 then I have this error:
```
root:DEBUG: js optimizer using native
splitting up js optimization into 14 chunks, using 4 cores  (total: 14.40 MB)
emscripten:DEBUG: Cache: PID 7331 released multiprocess file lock to Emscripten cache at /home/caiiiycuk/.emscripten_cache/asmjs
Traceback (most recent call last):
  File "/home/caiiiycuk/emscripten/sdk/emscripten/emcc.py", line 3075, in <module>
    sys.exit(run())
  File "/home/caiiiycuk/emscripten/sdk/emscripten/emcc.py", line 2050, in run
    optimizer.flush()
  File "/home/caiiiycuk/emscripten/sdk/emscripten/emcc.py", line 280, in flush
    just_concat='emitJSON' in chunk)
  File "/home/caiiiycuk/emscripten/sdk/emscripten/emcc.py", line 310, in run_passes
    output_filename=self.in_temp(os.path.basename(final) + '.jsopted.js'))
  File "/home/caiiiycuk/emscripten/sdk/emscripten/tools/shared.py", line 2334, in js_optimizer
    ret = js_optimizer.run(filename, passes, NODE_JS, debug, extra_info, just_split, just_concat)
  File "/home/caiiiycuk/emscripten/sdk/emscripten/tools/js_optimizer.py", line 577, in run
    return temp_files.run_and_clean(lambda: run_on_js(filename, passes, js_engine, source_map, extra_info, just_split, just_concat))
  File "/home/caiiiycuk/emscripten/sdk/emscripten/tools/tempfiles.py", line 100, in run_and_clean
    return func()
  File "/home/caiiiycuk/emscripten/sdk/emscripten/tools/js_optimizer.py", line 577, in <lambda>
    return temp_files.run_and_clean(lambda: run_on_js(filename, passes, js_engine, source_map, extra_info, just_split, just_concat))
  File "/home/caiiiycuk/emscripten/sdk/emscripten/tools/js_optimizer.py", line 470, in run_on_js
    filenames = pool.map(run_on_chunk, commands, chunksize=1)
TypeError: map() got an unexpected keyword argument 'chunksize'
ninja: build stopped: subcommand failed.
```

I think that problem is global, because in travis build I have exactly same error (based on docker trzeci/emscripten:sdk-incoming-64bit):
```
emscripten:DEBUG: Cache: PID 95 released multiprocess file lock to Emscripten cache at /emsdk_portable/data/.cache/asmjs
Traceback (most recent call last):
  File "/emsdk_portable/sdk/emcc.py", line 3075, in <module>
    sys.exit(run())
  File "/emsdk_portable/sdk/emcc.py", line 2050, in run
    optimizer.flush()
  File "/emsdk_portable/sdk/emcc.py", line 280, in flush
    just_concat='emitJSON' in chunk)
  File "/emsdk_portable/sdk/emcc.py", line 310, in run_passes
    output_filename=self.in_temp(os.path.basename(final) + '.jsopted.js'))
  File "/emsdk_portable/sdk/tools/shared.py", line 2334, in js_optimizer
    ret = js_optimizer.run(filename, passes, NODE_JS, debug, extra_info, just_split, just_concat)
  File "/emsdk_portable/sdk/tools/js_optimizer.py", line 577, in run
    return temp_files.run_and_clean(lambda: run_on_js(filename, passes, js_engine, source_map, extra_info, just_split, just_concat))
  File "/emsdk_portable/sdk/tools/tempfiles.py", line 100, in run_and_clean
    return func()
  File "/emsdk_portable/sdk/tools/js_optimizer.py", line 577, in <lambda>
    return temp_files.run_and_clean(lambda: run_on_js(filename, passes, js_engine, source_map, extra_info, just_split, just_concat))
  File "/emsdk_portable/sdk/tools/js_optimizer.py", line 470, in run_on_js
    filenames = pool.map(run_on_chunk, commands, chunksize=1)
TypeError: map() got an unexpected keyword argument 'chunksize'
CMakeFiles/wdosbox.dir/build.make:309: recipe for target 'wdosbox.js' failed
```

To reproduce:
* checkout https://github.com/caiiiycuk/js-dos/ (branch 6.22)
* build:
```
mkdir build
cd build
emcmake cmake -GNinja ..
EMCC_DEBUG=1 ninja -j4
```

--- a/tools/shared.py
+++ b/tools/shared.py
@@ -1416,13 +1416,13 @@ class Building(object):
       # actually spawn any new subprocesses. Very useful for internal debugging.
       if cores == 1:
         class FakeMultiprocessor(object):
-          def map(self, func, tasks):
+          def map(self, func, tasks, *args, **kwargs):
             results = []
             for t in tasks:
               results += [func(t)]
             return results
 
-          def map_async(self, func, tasks, **kwargs):
+          def map_async(self, func, tasks, *args, **kwargs):
             class Result:
               def __init__(self, func, tasks):
                 self.func = func

